agent/pkg/crane/k8s/status.go

```diff
@@ -9,16 +9,22 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	"github.com/dyrector-io/dyrectorio/agent/internal/util"
-	v1 "github.com/dyrector-io/dyrectorio/agent/pkg/api/v1"
 	"github.com/dyrector-io/dyrectorio/agent/pkg/crane/config"
 )
 
+type ContainerStatusResponse struct {
```

Comment:
Shouldn't this be in api.v1?

Reply:
This file is the only place where `ContainerStatusResponse` is used, also the `DeploymentStatus` function is unused, but I left it there for future possible use.

Reply:
We have `crux.GetDeployments` for for status queries, removing this is OK.